finality-aleph/src/data_io/chain_info.rs

```diff
@@ -197,7 +201,7 @@ where
         let mut curr_block = self.aux_finalized.clone();
         while curr_block.num > num {
             let parent_hash = self.chain_info_provider.get_parent_hash(&curr_block)?;
-            curr_block = (parent_hash, curr_block.num - NumberFor::<B>::one()).into();
+            curr_block = (parent_hash, curr_block.num - 1).into();
```

Comment:
nit: I would still use BlockNumber::one() or B::Header::Number::One() but thats just me :P

Reply:
And what is the reason for that? Right now here we have `BlockNumber = u32`. And if its type changes, compiler will just shout an error here. Also this type should never change. At least it does not have to in the next 100 years.